decidim-core/lib/decidim/core/test/factories.rb

```diff
@@ -747,12 +747,10 @@ def generate_localized_title
     amender { emendation.try(:creator_author) || emendation.try(:author) }
     state { "evaluating" }
 
-    trait :draft do
-      state { "draft" }
-    end
-
-    trait :rejected do
-      state { "rejected" }
+    Decidim::Amendment::STATES.keys.each do |defined_state|
+      trait defined_state.to_sym do
```

Comment:
Isn't it already a symbol? I think we can remove `.to_sym`.
```suggestion
      trait defined_state do
```